stable/mlrun-kit/values-minio.yaml

```diff
@@ -0,0 +1,202 @@
+      system:
+      - level: debug
+        sink: myStdoutLoggerSink
+      functions:
+      - level: debug
+        sink: myStdoutLoggerSink
+mlrun:
+  fullnameOverride: mlrun
+  nuclio:
+    mode: enabled
+  rbac:
+    create: true
+  v3io:
+    enabled: false
+  api:
+    fullnameOverride: mlrun-api
+    image:
+      tag: 1.0.3
+    service:
+      type: NodePort
+      nodePort: 30070
+    volumes:
+      storageOverride:
+        persistentVolumeClaim:
+          claimName: mlrun-pvc
+    extraEnv:
+      - name: MLRUN_HTTPDB__PROJECTS__FOLLOWERS
+        value: nuclio
+      - name: S3_ENDPOINT_URL
+        value: http://minio.mlrun.svc.cluster.local:9000
+      - name: AWS_SECRET_ACCESS_KEY
+        value: minio123
+      - name: AWS_ACCESS_KEY_ID
+        value: minio
+      - name: MLRUN_HTTPDB__REAL_PATH
+        value: s3://
+      - name: MLRUN_ARTIFACT_PATH
+        value: s3://mlrun/
+    extraPersistentVolumeMounts: ~
+  ui:
+    fullnameOverride: mlrun-ui
+    service:
+      type: NodePort
+      nodePort: 30060
+    image:
+      tag: 1.0.3
+
+# mlrun persistency resources creation can be disabled if the user would like to provide their own PVC
+mlrunPersistency:
+  pv:
+    create: true
+  pvc:
+    create: true
+
+sharedPersistency:
+  pvc:
+    create: false
+    spec:
+      storageClassName: nfs
+      accessModes:
+        - ReadWriteMany
+      resources:
+        requests:
+          storage: 1Gi
+
+jupyterNotebook:
+  name: jupyter-notebook
+  enabled: false
+  service:
+    type: NodePort
+    nodePort: 30040
+    port: 8888
+  ingress:
+    enabled: false
+    annotations: { }
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+    hosts:
+      - host: chart-example.local
+        paths: [ ]
+    tls: [ ]
+    #  - secretName: chart-example-tls
+    #    hosts:
+    #      - chart-example.local
+  image:
+    repository: quay.io/mlrun/jupyter
+    tag: 1.0.3
+    pullPolicy: IfNotPresent
+  # use this to override mlrunUIURL, by default it will be auto-resolved to externalHostAddress and
+  # mlrun UI's node port
+  mlrunUIURL:
+  persistency:
+    enabled: true
+    pvc:
+      create: true
+      spec:
+        storageClassName: nfs
+        accessModes:
+        - ReadWriteMany
+        resources:
+          requests:
+            storage: 1Gi
+
+mpi-operator:
+  nameOverride: mpi-operator
+  crd:
+    create: false
+  rbac:
+    clusterResources:
+      create: false
+    namespaced:
+      create: false
+  deployment:
+    create: false
+
+## customize PV spec used for nfs-server provisioner
+## ref: https://kubernetes.io/docs/concepts/storage/persistent-volumes/
+##
+nfsPv:
+  create: false
+  specOverride: {}
+
+nfs-server-provisioner:
+  replicaCount: 0
+  storageClass:
+    create: false
+  rbac:
+    create: false
+  nameOverride: nfs-server-provisioner
+  persistence:
+    enabled: false
+    storageClass: "mlrun-kit-nfs"
+    size: 5Gi

```

Comment:
I think we should add a comment stating that this is not supposed to be used because we use minio instead.